tasks/telemetry.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Build language telemetry map
+  set_fact:
+    telemetry_library_key:
+      java: specified_java_lib_version
+      js: specified_nodejs_lib_version
+      dotnet: specified_dotnet_lib_version
+      python: specified_python_lib_version
+      ruby: specified_ruby_lib_version
+
+- name: Build APM library installation payloads for telemetry
+  set_fact:
+    telemetry_payload:
+      "{{ telemetry_payload | default([]) |
```

Comment:
Should the `default` here provide a dict instead of list? `combine` is to be applied to dicts AFAICS.

Reply:
done